src/nassun_args.rs

```diff
@@ -15,6 +15,9 @@ pub struct NassunArgs {
     #[arg(from_global)]
     registry: Url,
 
+    #[arg(from_global)]
+    credentials: Vec<(String, String, String)>,
```

Comment:
you'll want to actually apply these in `to_nassun` below.

Reply:
Done.